test/local_integration_test.py

```diff
@@ -100,24 +100,29 @@ def wait_for_queue_level(self, empty: bool = True, timeout: int = 60):
         queues = {queue_name: boto3.resource('sqs').get_queue_by_name(QueueName=queue_name)
                   for queue_name in queue_names}
         wait_start_time = time.time()
+        queue_size_history = deque(maxlen=10)
 
         while True:
             total_message_count = self.get_number_of_messages(queues)
             queue_wait_time_elapsed = (time.time() - wait_start_time)
+            queue_size_history.append(total_message_count)
+            cumulative_queue_size = sum(queue_size_history)
             if queue_wait_time_elapsed > timeout:
                 logger.error('The queue(s) are NOT at the desired level.')
                 return
-            elif (total_message_count <= 0) == empty:
+            elif (cumulative_queue_size == 0) == empty:
                 logger.info('The queue(s) at the desired level.')
                 break
+            else:
+                logger.info('The most recently sampled queue sizes are %r.', cumulative_queue_size)
```

Comment:
```suggestion
                logger.info('The most recently sampled queue sizes are %r.', queue_size_history)
```

I did [ask](https://github.com/DataBiosphere/azul/pull/793#discussion_r265741718) for this in the last review.

Reply:
Sorry, I misread the change.